proofs/cbmc/poly_uniform_4x/poly_uniform_4x_harness.c

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) The mldsa-native project authors
+// SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+
+#include "poly.h"
+
+void harness(void)
+{
+  poly *r0;
+  uint8_t seed[4][MLD_ALIGN_UP(MLDSA_SEEDBYTES + 2)];
```

Comment:
Blocker: This must be `uint8_t *seed` -- otherwise the proof would even go through without memory preconditions for `seed`.

Reply:
Have you tried this?
```c
  poly *r0;
  uint8_t (*seed)[MLD_ALIGN_UP(MLDSA_SEEDBYTES + 2)];
  poly_uniform_4x(r0, seed);
```

Reply:
Your solution seems to work - thank you! Pushed it just now
I tried but failed to do this differently.